### PR TITLE
Update a few items in the table

### PR DESCRIPTION
**Pull request by ferdi05**
# Pull Request

## Related issue
Fixes #<issue_number>

## What does this PR do?
- ...

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: learn/what_is_meilisearch/comparison_to_alternatives.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 |   | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
 | Placeholder search | ✅ | ✅ | ✅ | ✅ |
-| Multi-index search | [2023](https://github.com/meilisearch/product/issues/74) | ✅ | ✅ | ✅ |
+| Multi-index search | [In beta](https://github.com/meilisearch/instant-meilisearch/releases/tag/v0.10.1-multi-index-search.0) | ✅ | ✅ | ✅ |
```
Comment (dichotommy) [inline learn/what_is_meilisearch/comparison_to_alternatives.md:98]: This line is the only part of this PR that I'm concerned about.

Communicating this Instant Meilisearch feature as beta multi-index search could be viewed as deceptive. Unless I'm mistaken, Meilisearch itself (engine/api) still does not support multi-index search. I don't know exactly what Instant Meilisearch is doing under the hood, but I'm guessing it's not "true" multi-index search, more likely duplicating the search query in an imitation of multi-index search that has some performance costs.

@bidoubiwa @irevoire What do you think about this?

Comment (bidoubiwa) [reply learn/what_is_meilisearch/comparison_to_alternatives.md:98]: I agree with @dichotommy.

When users are using Meilisearch through instantsearch.js, they are now able to use the `Index` widget. Which leads to enabling multi-index search in this scenario.

But nor Meilisearch nor the other SDK's provide that feature

Comment (ferdi05) [reply learn/what_is_meilisearch/comparison_to_alternatives.md:98]: Thanks for the clarification @dichotommy @bidoubiwa I will update this accordingly

**Review by dichotommy — APPROVED** [review]
LGTM!

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**